src/main/java/me/ryanhamshire/GriefPrevention/PlayerEventHandler.java

```diff
@@ -39,6 +39,7 @@
 import org.bukkit.block.BlockFace;
 import org.bukkit.block.data.BlockData;
 import org.bukkit.block.data.Waterlogged;
+import org.bukkit.enchantments.Enchantment;
```

Comment:
We can drop this import then too

Reply:
I keep forgetting about these lmao, sorry plugin dev isn't something I typically do